GPWP tries to include json files from externals

I import .json configuration files that my application is supposed to read at launch, so i included them in the webpack externals to not get them bundled
```
    externals: [
      /\/proxies\.json$/,
      /\/canvases\.json$/,
      /^\.\/styleassets\.json$/,
      /^\.\/assets\.json$/,
      nodeExternals(),
    ],

```
but the GPJWP plugin, after a recent update, wants to include them and that gives some warnings:

```
GPJWP: Ignoring module without a found package.json: . ("package.json" couldn't resolve): Cannot find module '/home/andreas/Projekte/pixelplanet/server.js'. Please verify that the package.json has a valid "main" entry
GPJWP: Couldn't resolve a version for module "." (from portable ID: external commonjs2 "./canvases.json") - will be ignored in final output.
GPJWP: Ignoring module without a found package.json: . ("package.json" couldn't resolve): Cannot find module '/home/andreas/Projekte/pixelplanet/server.js'. Please verify that the package.json has a valid "main" entry
GPJWP: Couldn't resolve a version for module "." (from portable ID: external commonjs2 "./assets.json") - will be ignored in final output.
GPJWP: Ignoring module without a found package.json: . ("package.json" couldn't resolve): Cannot find module '/home/andreas/Projekte/pixelplanet/server.js'. Please verify that the package.json has a valid "main" entry
GPJWP: Couldn't resolve a version for module "." (from portable ID: external commonjs2 "./styleassets.json") - will be ignored in final output.
GPJWP: Ignoring module without a found package.json: . ("package.json" couldn't resolve): Cannot find module '/home/andreas/Projekte/pixelplanet/server.js'. Please verify that the package.json has a valid "main" entry
GPJWP: Couldn't resolve a version for module "." (from portable ID: external commonjs2 "./proxies.json") - will be ignored in final output.
```

This didn't happen before, so i thought that might be a bug :)

--- a/index.js
+++ b/index.js
@@ -226,6 +226,16 @@ GeneratePackageJsonPlugin.prototype.apply = function (compiler) {
       logIfDebug(`GPJWP: Found external module: ${portableId}`);
       const moduleName = getNameFromPortableId(portableId);
 
+      if (moduleName === '.' || moduleName === '..') {
+        logIfDebug(`GPJP: excluded "${portableId}" because it is on a relative path`);
+        return;
+      }
+
+      if (this.excludeDependencies.includes(moduleName)) {
+        logIfDebug(`GPJWP: excluded "${moduleName}" from generated package.json`);
+        return;
+      }
+
       if (moduleName.length === 0) {
         console.error(`GPJWP: Couldn't decipher the module name from external module input: ${portableId} - will be ignored in final output.`);
         return;
@@ -331,13 +341,6 @@ GeneratePackageJsonPlugin.prototype.apply = function (compiler) {
             }
           }
         }
-      }
-    }
-
-    for (const depName of this.excludeDependencies) {
-      if (modules[depName] != null) {
-        delete modules[depName];
-        logIfDebug(`GPJWP: excluded "${depName}" from generated package.json`);
       }
     }
 

